datastore/__init__.py

```diff
@@ -165,22 +207,130 @@ async def upload_picture(cursor, user_id, picture_hash, container_client):
         response = await azure_storage.upload_image(
             container_client, "General", picture_hash, picture_id
         )
-        if not response:
-            raise BlobUploadError("Error uploading the picture")
         # Update the picture metadata in the DB
         data = {
             "link": "General/" + str(picture_id),
```

Comment:
Since we are implementing the usage of selecting the picture_set_id, the link might not be General/. It should be the picture_set_id with our current architecture. However I see the increase into the need of implementing a proper structure to name the folder in the Datastore, I suggest this to be a specific and new issue